frontend/src/app/reducers/browser.js

```diff
@@ -1,9 +1,12 @@
 import { handleActions } from 'redux-actions';
 
 const setIsFirefox = (state, { payload: isFirefox }) => ({ ...state, isFirefox });
+const setIsMinFirefox = (state, { payload: isMinFirefox }) => ({ ...state, isMinFirefox });
 
 export default handleActions({
-  setIsFirefox
+  setIsFirefox,
+  setIsMinFirefox
 }, {
-  isFirefox: false
+  isFirefox: false,
+  isMinFirefox: false
```

Comment:
@chuckharmston once i figured this out i was able to knock out the browser actions file and remove the dispatches from index.